pudb/debugger.py

```diff
@@ -2504,7 +2504,10 @@ def event_loop(self, toplevel=None):
                     if k == "window resize":
                         self.size = self.screen.get_cols_rows()
                     else:
-                        toplevel.keypress(self.size, k)
+                        try:
+                            toplevel.keypress(self.size, k)
+                        except Exception:
+                            ui_log.exception("Encountered unexpected exception in the debugger UI.")
```

Comment:
Maybe add some text telling people to look in the console window for the exception text?

Reply:
Where would I put the text? In a popup?

Reply:
Ah, nvm. This message itself isn't very visible. Fair, we can just bury it in the UI log for now. It's not likely to be very interesting to end users anyway.

Reply:
Hmm, OTOH. I think this merits a popup, because likely that means whatever the user asked us to do didn't happen.

#419